include/NotificationsManager.php

```diff
@@ -213,8 +213,9 @@ private function formatNotifs($notifs, $ident = "") {
 				// Because we use different db tables for the notification query
 				// we have sometimes $it['unseen'] and sometimes $it['seen].
 				// So we will have to transform $it['unseen']
-				if($it['unseen'])
+				if (array_key_exists('unseen', $it)) {
```

Comment:
There is a difference `array_key_exists()` does respect `NULL` but in this case this shouldn't make a difference.